app/globals.css

```
@@ -1,58 +1,276 @@
+}
+
+.landing-page::-webkit-scrollbar-thumb:hover {
+  background-color: #3B82F6;
+}
+
+.landing-page {
+  scrollbar-color: #93C5FD #f1f5f9;
+}
+
+/* ─── Pipeline Flow Animation ─── */
+@keyframes flowPulse {
```

Stylelint (17.9.0)

[error] Expected keyframe name "flowPulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)